### ISSUE TITLE
Tabindex isn't passed to cloned nav items

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5561]
**Describe the issue. What is the expected and unexpected behavior?**
When products use their own nav items, the tabIndex isn't added to the cloned child so [this PR](https://github.com/patternfly/patternfly-react/pull/5312) wasn't affecting them. The tabIndex should be added so that keyboard navigation will function properly.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug 

**What is your product and what release version are you targeting?**
Noticed in OpenShift nav
